src/libsyntax/feature_gate.rs

```diff
@@ -525,63 +525,73 @@ declare_features! (
 );
 
 declare_features! (
-    (accepted, associated_types, "1.0.0", None, None),
```

Comment:
why hae these `1.0.0` features been reordered?

Reply:
`issue_5723_bootstrap` and `test_accepted_feature` are "fake features" so I grouped them separately from the real features; I also moved `if_let` to be colocated with `while_let`.